### DISCUSSION TITLE
[Question]: Track custom storable with UndoManager

### DISCUSSION THREAD

**Discussion by rhoenerSBS** [GrapesJS/grapesjs#6178] [Q&A]
### GrapesJS version

- [X] I confirm to use the latest version of GrapesJS

### What browser are you using?

Chrome v129

### Reproducible demo link

not applicable

### Describe the bug

HI @artf,

I am currently implementing an editor with grapesjs in which it is possible to add chapters alongside the already existing pages. I implemented a ChapterStorage that implements the IStorableModule to store the chapters in the project data.
My question is what is the best way to let the UndoManager track the changes made to the chapters?
Is it necessary to extend the ItemManagerModule in some way?

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by artf** [comment]
Hey @rhoenerSBS please avoid opening bug issues for questions.

**Reply by rhoenerSBS** [reply]
Hey @artf,
sorry I will avoid it from now on. Could you help me with my question?
